packages/app-desktop/commands/restoreNoteRevision.ts

```diff
@@ -1,9 +1,10 @@
 import { CommandRuntime, CommandDeclaration, CommandContext } from '@joplin/lib/services/CommandService';
 import RevisionService from '@joplin/lib/services/RevisionService';
+import { _ } from '@joplin/lib/locale';
 
 export const declaration: CommandDeclaration = {
 	name: 'restoreNoteRevision',
-	label: 'Restore a note from history',
+	label: () => _('Restore a note from history'),
```

Comment:
We won't translate this, it's not officially supported

Reply:
No worries, I reverted the change to that file and updated the PR